Sources/SwiftHtml/Attributes/Global.swift

```diff
@@ -39,20 +39,23 @@ private extension String {
     }
 }
 
-private extension Array where Element == String {
+private extension Set where Element == String {
     
-    var classString: String {
-        filter { !$0.isEmpty }.joined(separator: " ")
-    }
+    var classString: String { joined(separator: " ") }
```

Comment:
Filtering empty strings is handled just once when setting attribute value... so this is redundant